Packs/Recorded_Future/Integrations/Recorded_Future/Recorded_Future.yml

```diff
@@ -220,23 +297,56 @@ script:
     - contextPath: File.RecordedFuture.LastSeen
       description: Risk last seen timestamp
       type: date
-    description: Returns threat intelligence information for a file in Recorded Future.
-  - name: recorded-future-get-related-entities
-    arguments:
-    - name: entityType
-      required: true
-      auto: PREDEFINED
+    - contextPath: File.Tags
+      description: Tags that are associated with the Domain
+      type: String
+    - contextPath: File.CommunityNotes.Note
+      description: '...'
```

Comment:
A summary note on the mentioning of the file in a source monitored by Recorded Future.